src/content/smart-bulletins/037.md

```diff
@@ -0,0 +1,112 @@
+company.
+
+OMB Memo M-23-13 mainly addresses requirements for agencies to identify use of
+TikTok, remove any instances of its use either within an agency or by agency contractors,
+ensure contracts requiring the use of IT do not involve use of this prohibited technology,
+and provides a timeline for these actions to be completed, assuming no exception for
+authorized use is applicable.
+
+FAR case 2023-010 is an interim final rule that became effective upon the June 2, 2023
+date of publication. In summary, it addresses contract clause requirements to implement
+the TikTok ban on formal contracts. It also states that the prohibition is applicable to
+actions within the simplified acquisition threshold, including micro-purchases.
+
+## Summary
+
+Every year, organizations participating in the GSA SmartPay program use purchase cards
+to obtain approximately $23 billion in products and services in support of their missions
+through approximately 19 million purchase transactions.
+
+The Act addresses TikTok and any successor application or service that is developed or
+provided by ByteDance Limited and prohibits their use by “executive agencies” as defined
+in 41 U.S.C § 133.2 and in all IT as defined in 40 U.S.C. § 11101(6). Exceptions to this
+prohibition are addressed in OMB memo M-23-13.
+
+Purchases made at or below the micro-purchase threshold are subject to the Act unless
+an exception is granted in accordance with the OMB memo (i.e., for national security, law
+enforcement, or security research activities).
+
+This prohibition is narrow (e.g., affecting social media or advertising services; purchases
+of services that require use of IT on which TikTok may be present) and it is not anticipated
+to affect many purchase cardholder micro-purchase transactions.
+
+It is important to note that the definition of IT used for this ban encompasses not only IT
+owned or operated by agencies, but also “used by a contractor under a contract with the
+executive agency that requires the use” of that IT, whether expressly or “to a significant
+extent in the performance of a service or the furnishing of a product” (M-23-13 at 1-2).
+That definition does not, however, “include any equipment acquired by a federal contractor
+incidental to a federal contract.”
+
+## Action
+
+GSA SmartPay purchase cards shall not be used for purchases that require the use of
+TikTok or require vendor use of IT that contains TikTok, unless an exception is granted in
+accordance with Office of Management and Budget Memorandum M-23-13. Based on a
+Center for Charge Card Management review, agency purchase cardholders rarely acquire
+media services or make purchases through TikTok. As a result, very few purchase card
+transactions (primarily social media advertising and advertising services) are expected to
+be impacted by this prohibition.
+
+However, agencies which routinely purchase social media and advertising services within
```

Comment:
However, agencies that routinely purchase social media and advertising services within the micro-purchase threshold using a purchase card may consider issuing a policy that limits buying these services on the open market to help manage this risk. As a best practice, agencies may also consider issuing policy emphasizing the security benefits of using existing government online shopping sites as the first source of supply for high-risk products and services, such as GSA Advantage!, Fed Mall, GSA Global Supply, category-managed solutions, agency-specific vehicles, etc. The expectation is that the contracting office will ensure the required TikTok prohibition is incorporated into the contracts underlying these offerings.

Reply:
this replaces the content from line 85 through 93